sdk/rust/oak/src/storage/mod.rs

```diff
@@ -88,19 +78,37 @@ impl Storage {
         let mut grpc_request = GrpcRequest::new();
         grpc_request.method_name = grpc_method_name.to_owned();
         grpc_request.set_req_msg(request_any);
+        let mut grpc_data = Vec::new();
+        grpc_request.write_to_writer(&mut grpc_data).unwrap();
+
+        // Create an ephemeral channel for the response.
+        let (rsp_out, rsp_in) = match crate::channel_create() {
```

Comment:
Is it possible to use https://doc.rust-lang.org/edition-guide/rust-2018/error-handling-and-panics/the-question-mark-operator-for-easier-error-handling.html here? I don't remember if we ever implemented `Into` or `From` between status and `grpc::Result`.

Reply:
We didn't implement `From` (IIRC it was tricky because neither source nor destination types are in a modifable crate)